frontend/src/metabase/dashboard/actions/tabs.ts

```diff
@@ -322,6 +349,28 @@ export const tabsReducer = createReducer<DashboardState>(
       },
     );
 
+    builder.addCase(
+      undoMoveDashCardToTab,
+      (state, { payload: { dashCardId } }) => {
+        if (!state.dashboardId) {
+          throw Error(`state.dashboardId is null in undoMoveDashCardToTab`);
+        }
```

Comment:
Is there a case where this could happen?

Reply:
same as other comment